step.sh

```diff
@@ -97,7 +97,7 @@ echo_details "* export_method: $export_method"
 echo_details "* upload_bitcode: $upload_bitcode"
```

Comment:
if `custom_export_options_plist_content ` provided: `ignoring this options because custom_export_options_plist_content  provided, the ignored value: xyz`